user/avatar.go

```diff
@@ -0,0 +1,11 @@
+package user
+
+import (
+	"net/url"
+	"strconv"
+)
+
+// GetAvatarURL returns the URL to the avatar with the given userID and size in pixels
+func (t *TalkUser) GetAvatarURL(userID string, size int) (avatarURL string) {
+	return t.NextcloudURL + "/index.php/avatar/" + url.PathEscape(userID) + "/" + strconv.Itoa(size)
```

Comment:
Can you enforce a "power of 2" pixel size? 32, 64, ...

Reply:
I had not realized that was preferred. Will do :+1:

Reply:
Fine either way, although we will mostlikely just enforce it on the api itself at somepoint, so would be cool already to train your users